[Docs-2.0] Update `ContributionChecklist.md` (#1038)
zkSNACKs/WasabiDoc: docs/building-wasabi/ContributionChecklist.md

So you're interested in contributing to Wasabi - welcome!
This checklist will get you plugged in and productive as quickly as possible.

> :::tip Embrace the open-source cryptoanarchy!

Reviewer: ```suggestion
:::tip Embrace the open-source crypto-anarchy!
```

It is not required to work on an existing issue to contribute to Wasabi, and no one is here to tell you what to do.

> Contributors who have their own ideas are free to work in their own forks on whatever they wish, however they wish, and without any permission from anyone.

Reviewer: ```suggestion
Contributors who have their own ideas are free to work at their own forked version on whatever they wish, however they wish and without any permission from anyone.
```

> It is not required to work on an existing issue to contribute to Wasabi, and no one is here to tell you what to do.
> Contributors who have their own ideas are free to work at their own forked version on whatever they want, however they wish and without any permission from anyone.

Reviewer 2: How about this:

```suggestion
No one is here to tell you what to do, and it is not required to work on an existing issue to contribute to Wasabi.
Contributors are free to work on their own forked versions on whatever they want, however they wish and without any permission from anyone.
```

With that being said, in order to maximize the usefulness/impact of your work, it's a good idea to consult with other Wasabikas via Slack, GitHub, or other communication channels before setting out on any serious contribution effort.
:::

[[toc]]

---

## Who is a contributor?

> 

Reviewer 2: Why removing these empty lines?
We have them in all documents so we better be consistent and keep them after the titles.

Author: Consistency was what I was aiming for. I'll add a space then

This will help others point you in the right direction.
- Explore the communication channels and find out what the peers are tinkering on, learn about the project and who is contributing in what way.
This will help you to find the interesting challenges you can work on.

> - Follow [@WasabiWallet](https://twitter.com/wasabiwallet) on Twitter and subscribe to the [Wasabi YouTube channel](https://www.youtube.com/channel/UCobsrSexTuVkL39mbrQ35VQ) to stay up-to-date.

Reviewer 2: Why this got removed?

Author: Didnt see following Wasabi Twitter or Youtube as necessary for new contributors.

Everyone is welcome to join the discussions. 
However, being familiar with the topic (if there is any) is recommended.
Discussions might be recorded and published on the Wasabi Wallet [YouTube channel](https://www.youtube.com/c/wasabiwallet).
You can find the schedule, call link, past recordings and more at the [Wasabi Research Club repository](https://github.com/zkSNACKs/WasabiResearchClub/).

## Learn how Wasabikas work
- Educate yourself about the software by reading through [Using Wasabi](/using-wasabi/).

> - Learn what Wasabikas have planned for the [future Of Wasabi Wallet](/building-wasabi/FutureOfWasabiWallet.md).

Reviewer 2: At this point there is no such document.

Author: Can we leave it as a placeholder?
It was there to remind me.

- To understand how Wasabi CoinJoins work, read [WabiSabi Framework](https://eprint.iacr.org/2021/206).
- Familiarize yourself with [C4: The Collective Code Construction Contract](https://rfc.unprotocols.org/spec:1/C4/).
It’s a simple set of collaboration rules based on GitHub’s fork and pull request model, and a foundational part of how we work together.

## What to work on
Education and development are vitally important aspects of Bitcoin privacy and Wasabi Wallet specifically.
The more Wasabi users there are, the better their privacy, because [anonymity loves company](https://www.freehaven.net/anonbib/cache/usability:weis2006.pdf).

> The better UX and privacy Wasabi provides, the more there will be users, because of [network effect](https://en.wikipedia.org/wiki/Network_effect).

Reviewer 2: Do you want these to be on separate lines? If yes then you need to separate them by an empty line or better to use `-` at the start of each line.

### Education Contributor
1. **Videos, podcasts, memes and articles** are great ways of sharing information to your peers.

> You can create wallet tutorial videos, insightful recorded discussions, articles or memes about Wasabi and privacy.

Reviewer: ```suggestion
You can create wallet tutorials, record insightful discussions, write articles or make memes about Wasabi and privacy.
```

2. **Organizing events and doing presentations** about Wasabi Wallet and Bitcoin privacy.
Connecting like-minded individuals, who want to take control of their own financial privacy and teaching them about the awesome tools that are available, can be life changing.

> 3. **Helping and teaching new users**, once you are familiar with the software, is something that can't be underestimated.

Reviewer: ```suggestion
3. **Helping and teaching new users**, being familiar with the software is something that can't be underestimated.
```

You can do this in any social media like Twitter, Telegram, Reddit etc or in your own neighborhood.

> :::tip [zkSNACKs Ltd.](https://zksnacks.com/) is known for compensating some of the educators when they consistently help out new users or create good content. 

Reviewer: ```suggestion
:::tip [zkSNACKs Ltd.](https://zksnacks.com/) is known for compensating educators who consistently help out new users or create good content. 
```

The company is actively sponsoring Bitcoin events and organizing [Contribution Games](https://blog.wasabiwallet.io/coming-soon-contribution-games/).

> In case you have been contributing already, feel free to let other Wasabikas know by sending us a link in [Slack](https://join.slack.com/t/tumblebit/shared_invite/enQtNjQ1MTQ2NzQ1ODI0LWIzOTg5YTM3YmNkOTg1NjZmZTQ3NmM1OTAzYmQyYzk1M2M0MTdlZDk2OTQwNzFiNTg1ZmExNzM0NjgzY2M0Yzg).

Reviewer 2: Which link?

Author: Link to their content but I guess that can be explained a bit better.
